qt/editor_dialog.cpp

```diff
@@ -113,12 +114,12 @@ EditorDialog::EditorDialog(QWidget * parent, osm::EditableMapObject & emo)
       {
         grid->addWidget(new QLabel(kInternetObjectName), row, 0);
         QComboBox * cmb = new QComboBox();
-        std::string const values[] = {DebugPrint(osm::Internet::Unknown),
-                                      DebugPrint(osm::Internet::Wlan),
-                                      DebugPrint(osm::Internet::Wired),
-                                      DebugPrint(osm::Internet::Terminal),
-                                      DebugPrint(osm::Internet::Yes),
-                                      DebugPrint(osm::Internet::No)};
+        std::string const values[] = {DebugPrint(feature::Internet::Unknown),
```

Comment:
Can it be `std::string_view constexpr values[]` array?

Reply:
DebugPrint will return string